.github/workflows/nv-transformers-v100.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install pytorch
         run: |
           # use the same pytorch version as transformers CI
-          pip install -U --cache-dir $TORCH_CACHE torch==2.0.1+cu118 --index-url https://download.pytorch.org/whl/cu118
+          pip install -U --cache-dir $TORCH_CACHE torch==2.1.0+cu118 --index-url https://download.pytorch.org/whl/cu118
```

Comment:
I think the underlying error we get when updating to this might be coming from the fact that we are using cuda 116 here, but I'm not certain.